e2e/test/scenarios/embedding-sdk/create-question.cy.spec.ts

```diff
@@ -20,9 +20,17 @@ describeSDK("scenarios > embedding-sdk > create-question", () => {
     cy.signOut();
   });
 
-  it("can create questions via the CreateQuestion component", () => {
+  it("can create a question via the CreateQuestion component", () => {
     cy.intercept("POST", "/api/card").as("createCard");
 
+    cy.intercept("POST", "/api/dataset", req => {
+      // throttling makes the loading indicator shows up
+      // reliably when we click on visualize.
+      req.on("response", res => {
+        res.setThrottle(1000);
+      });
```

Comment:
I couldn't think of another way to reduce test flakiness when checking for loading indicators, let me know if you have a better idea 🙏🏻